test/bigtest/tests/export-to-CSV-test.js

```diff
@@ -80,35 +80,10 @@ describe('Export to CSV', () => {
     describe('Export expired holds to CSV', function () {
       beforeEach(async function () {
         await requests.headerDropdown.click();
-        await requests.headerDropdownMenu.clickExportExpiredHoldsToCSV();
       });
 
       it('exports data to csv', () => {
-        expect(requests.headerDropdownMenu.exportExpiredHoldsBtnIsVisible).to.be.false;
-      });
-
-      describe('error modal ', () => {
```

Comment:
Could you please clarify the reason for removing tests for error modal?

Reply:
This modal was removed from component and from tests as well, because of the **Export hold shelf clearance report** should be disabled if there is nothing on this report for the currently selected service point, we do not need this error modal any more, and also it will never show because we cannot click to disabled link